### PR TITLE
Added Configurations Menu. Updated SNAP menu.

### PR DESCRIPTION
**Pull request by rose0531**
I added the configurations menu. You can adjust some of the settings via sliders and input fields. Configuration settings are not currently saved into another file yet, this is just getting the menu system and layout ready. I also updated the main SNAP menu.

### CODE REVIEW

File: Assets/Scenes/Main Menu/MenuUI.cs [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	public void OpenConfigMenu(){
+		ConfigurationMenuUI.SetActive (true);
+		Time.timeScale = 0f;
+		NewConfigurationMenuIsOpen = true;
+		EscapeMenuIsOpen = false;
+	}
+	/****************************************************************************************************/
+
+
+	/********************************* Frequency Configuration ******************************************/
```
Comment (dustin-fox) [inline Assets/Scenes/Main Menu/MenuUI.cs]: Excellent code documentation

**Comment by dustin-fox** [comment]
Very nice start. The sliders are smooth and it all seems to work well. I only have two suggestions\concerns:
* The background does not scale with the buttons when you try it in full screen.
* This is not required but It would be nice if there was a "save" button or something that takes you from the config menu to the main menu again.

**Comment by rose0531** [comment]
I forgot lock the menu system in place. I'll do that so it'll scale when you full screen. I'll also add a save button.

**Comment by rose0531** [comment]
I closed the pull request while I add more to my branch.

**Pull request closed without merging**